Source/DafnyCore/Compilers/SinglePassCompiler.cs

```diff
@@ -347,6 +347,7 @@ protected interface IClassWriter {
     protected virtual string True { get => "true"; }
     protected virtual string False { get => "false"; }
     protected virtual string Conj { get => "&&"; }
+    protected virtual string AssignmentSymbol { get => " = "; }
```

Comment:
Ahah that's a fun fact that Dafny is the first language we support to override that AssignmentSymbol ^^. I had advocated to change that but I guess it's too late.